docs/dev-guides/7-xrpackage-api.md

```diff
@@ -170,9 +190,21 @@ _Retrieve the collision mesh of the XRPackage if it exists._
 
 ## `grabrelease`
 
-## `isAttached`
+## `isAttached()`
 
-## `load`
+**Parameters**: None
+
+**Returns**: a `Boolean` representing whether the current package is attached to an `XRPackageEngine` instance.
+
+## `load()`
```

Comment:
I think this is actually an internal method; it's probably never manually called.

And that could probably be marked in the package implementation itself. JS doesn't support private fields anywhere AFAIK but we could use an underscore.

Reply:
Thanks, updated all of these! I will make a PR to prepend these methods with an underscore in `xrpackage` later!